sf_examples/swarm_rl_examples/runs/multi_drone.py

```diff
@@ -0,0 +1,24 @@
+from sample_factory.runner.run_description import Experiment, ParamGrid, RunDescription
+from sf_examples.swarm_rl_examples.runs.quad_multi_mix_baseline import QUAD_BASELINE_CLI
+
+_params = ParamGrid(
+    [
+        ("seed", [0000, 1111, 2222, 3333]),
+    ]
+)
+
+SMALL_MODEL_CLI = QUAD_BASELINE_CLI + (
+    " --train_for_env_steps=10000000000 --num_workers=36 --num_envs_per_worker=4 "
+    "--quads_num_agents=8 --save_milestones_sec=10000 --async_rl=True --num_batches_to_accumulate=8 "
```

Comment:
Is there a good reason to keep --num_batches_to_accumulate=8?
Can't we use the default 2?

Reply:
Large value for this parameter will increase the policy lag.

Reply:
I'll change it back to 2. I think this was from when I was testing the freezing issue from there being more training batches compared to learner batches but this didn't help the issue.